src/SearchCount/__tests__/SearchCount.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
+import { SearchResults } from 'react-instantsearch-core';
+import { SearchCount } from '..'
+
+describe('Search Count Description', function () {
+  describe('With no results', function () {
+    const testSearchResults = {
+      nbHits: 0,
+      query: 'nothing',
+    } as SearchResults;
+    it('should show "0 results for:"', function  () {
+      render(<SearchCount searchResults={testSearchResults} />);
+      expect(screen.getByText('0 results for:')).toBeInTheDocument();
+    });
+  });
+  describe('With one result', function () {
+    const testSearchResults = {
+      nbHits: 1,
+      query: 'one thing',
+    } as SearchResults;
+    it('should show "1 result for:"', function  () {
+      render(<SearchCount searchResults={testSearchResults} />);
+      expect(screen.getByText('1 result for:')).toBeInTheDocument();
+    });
+  });
+  describe('With more than one result', function () {
+    const testSearchResults = {
+      nbHits: 42,
+      query: 'everything',
+    } as SearchResults;
+    it('should show "n results for:"', function  () {
+      render(<SearchCount searchResults={testSearchResults} />);
+      expect(screen.getByText('42 results for:')).toBeInTheDocument();
+    });
+  });
+  describe('With a blank query', function () {
+    const testSearchResults = {
+      nbHits: 42,
+      query: '',
+    } as SearchResults;
+    it('should show ""', function  () {
```

Comment:
I was splitting the tests for the "_n_ results for:" section away from the section that shows the query that was searched, since they're separate `<p>` elements, so this particular test isn't really concerned with how many results. I did add a little more clarification that it's specifically looking for "" in the query area in c6ac672